### PR TITLE
chore(RHTAPWATCH-404): update probe

### PR DESCRIPTION
**Pull request by avi-biton**
Update probe to select only from the latest grafana api 
Fix lint issues

### CODE REVIEW

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml [hunk 1]
```diff
@@ -48,19 +48,27 @@ spec:
...
+              requests:
+                cpu: 100m
+                memory: 10Mi
+              limits:
+                cpu: 100m
+                memory: 100Mi
+            securityContext:
+              readOnlyRootFilesystem: true
+              runAsNonRoot: true
+
```
Comment (yftacherzog) [inline config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:74]: Newline at end of file

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml [hunk 2]
```diff
@@ -11,7 +11,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: read-grafana
 rules:
-- apiGroups: ['grafana.integreatly.org']
+- apiGroups: ['grafanas.grafana.integreatly.org']
```
Comment (hugares) [inline config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml:14]: Should't the api group be `grafana.integreatly.org` and the resource is `grafanas`

**Review by yftacherzog — COMMENTED** [review]
Would it fail the job if grep comes empty?

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml (hunk 2 above)
Comment (avi-biton) [reply config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml]: Yes, it is leftover from my tries :)

**Comment by avi-biton** [comment]
> Would it fail the job if grep comes empty?

Yes, I tested it

**Review by yftacherzog — APPROVED** [review]

File: config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml (hunk 2 above)
Comment (hugares) [reply config/probes/monitoring/grafana/base/grafana-datasource-probe.yaml]: looks like I do not have the rights to resolve the comment thread I created... You will have to do it for me :)

**Pull request merged**